src/transformers/typescript.ts

```diff
@@ -53,6 +54,45 @@ const importTransformer: ts.TransformerFactory<ts.SourceFile> = (context) => {
   return (node) => ts.visitNode(node, visit);
 };
 
+function stripTags(markup: string): string {
+  return markup
+    .replace(/<!--[^]*?-->|<style(\s[^]*?)?(?:>([^]*?)<\/style>|\/>)/gi, '')
+    .replace(/<!--[^]*?-->|<script(\s[^]*?)?(?:>([^]*?)<\/script>|\/>)/gi, '');
+}
+
+function appendVarsToCode({
+  content,
+  markup,
+  filename,
+}: {
+  content: string;
+  markup?: string;
+  filename?: string;
+}): string {
+  if (!markup) return content;
+
+  const { vars } = compile(stripTags(markup), {
```

Comment:
Since the Svelte compiler will throw errors on unknown properties, this needs to be surrounded with a try-catch for backwards-compatibility.

Reply:
If I'm correct, it will depend on the release of `svelte`. If the 3 PRs are merged and released in the same time, there will be no problem since the `markup` will not be passed so it will not try to compile it.

Am I correct?

Reply:
Ah right, yes that is correct. Should be documented in the code then just to be sure for future readers.

Reply:
Maybe we should inform `svelte` maintainers that the 3 PRs should be released together.
Do you agree?

*If it's not possible, using a try-catch will solve the issue*